.gitignore

```diff
@@ -3,6 +3,7 @@ bundle
 tmp/
 tests/source/
 dist/
+.idea
```

Comment:
It is used so the jetbrains config get ignored, I can remove it if you feel its not neccessary